vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -140,6 +140,12 @@ installFIPS() {
     echo "Installing FIPS..."
     wait_for_apt_locks
 
+    # installing fips kernel doesn't remove non-fips kernel now, purge current linux-image-azure
+    echo "purging linux-image-azure..."
+    apt_get_purge 5 10 120 linux-image-azure
+    retrycmd_if_failure 120 5 25 apt-mark hold linux-image-azure
```

Comment:
none of these functions exit on failure and we don't `set -e` by default, consider adding `|| exit 1` or similar to force these cases